### PR TITLE
Fix grouped select

### PR DESCRIPTION
**Pull request by Minglee01**
@incuna/js please review.

Handle the fields toggling the other one

### CODE REVIEW

**Comment by wytrych** [comment]
Could you write a test for this change?

**Comment by Minglee01** [comment]
im not sure how to write this test

**Comment by wytrych** [comment]
Start with recreating the bug in the test

**Comment by wytrych** [comment]
Hmm it seems there are no tests for this particular directive. In that case maybe leave it.

File: dist/angular-input-fields.js [hunk 1]
```diff
@@ -137,7 +137,11 @@
 
                     // Set group when model changes.
                     scope.$watch('model', function (value) {
-                        if (angular.isDefined(value) && value !== null) {
+                        // When the value is null set the set box display to select.
+                        if (value === null) {
+                            scope.model = '';
```
Comment (wytrych) [inline dist/angular-input-fields.js]: This will trigger another `$watch`

**Comment by Minglee01** [comment]
@wytrych  change made
thanks

**Pull request merged**